### PR TITLE
Creating skeleton loader

### PR DESCRIPTION
**Pull request by diegolarraz**
## Objective
What: Skeleton loader
Why: Because the application is so fast that we need placeholders (which you can barely see) instead of animations as animations slow the UI down

## Type

- [x] New Feature

## Tested

Localhost

### CODE REVIEW

**Comment by HugoWith** [comment]
Hellooo
Well done guys @diegolarraz @BenDanielGreep 

What do you think of having it for the facets and the federated search ? 😊

File: src/components/searchresultpage/srpLaptop/SrpLaptop.jsx [hunk 1]
```diff
@@ -102,15 +101,15 @@ const SrpLaptop = () => {
       {/* Render Recommend component - Trending Products Slider */}
       {/* Change header and maxRecommendations in /config/trendingConfig.js */}
       <div className="recommend">
-        {shouldHaveTrendingProductsValue && queryState === "" && state?.type !== 'context' && (
+        {false && queryState === "" && (
```
Comment (matthew1809) [inline src/components/searchresultpage/srpLaptop/SrpLaptop.jsx:104]: why false, what is false? Can you please comment code which might not make sense to a new reader? Thanks!

File: src/components/trending/TrendingFacetValues.jsx [hunk 2]
```diff
@@ -64,17 +70,24 @@ const TrendingFacetValues = ({ items, refine }) => {
...
-                    <TrendingFacetsItem trendingFacetValue={trendingFacetValue} />
+                recommendations.map((trendingFacetValue, i) => {
+                  return  (
+                    recommendationsLoaded ? (
+                      <TrendingFacetsItem 
+                        trendingFacetValue={trendingFacetValue} 
+                        key={`${i}${trendingFacetValue}`} 
+                      />
+                    ) : (
+                      <div className="" key={i + "loader"} style={{width: "200px", height: "300px", backgroundColor: "green"}}></div>
```
Comment (matthew1809) [inline src/components/trending/TrendingFacetValues.jsx:89]: maybe move this to a CSS class?

File: src/scss/searchresultspage/searchresultspage.scss [hunk 3]
```diff
@@ -137,25 +137,25 @@
...
-  opacity: 0;
+  opacity: 1;
   visibility: hidden;
 }
 
 .srp-active {
-  opacity: 0;
-  animation: fadeIn 3s forwards cubic-bezier(0.43, 0.13, 0.23, 0.96);
+  opacity: 1;
+  // animation: fadeIn 3s forwards cubi÷c-bezier(0.43, 0.13, 0.23, 0.96);
```
Comment (matthew1809) [inline src/scss/searchresultspage/searchresultspage.scss:146]: Why comment and not remove?

**Review by matthew1809 — CHANGES_REQUESTED** [review]
Nice work!

**Comment by BenDanielGreep** [comment]
> Hellooo Well done guys @diegolarraz @BenDanielGreep
> 
> What do you think of having it for the facets and the federated search ? 😊

We do have it for the facets (on SRP)

We talked about this - in our opinion it is so fast that no one will need the skeleton on FS. 
However for the fringe case that the SE computer is running slow, we could implement the skeleton on FS on another sprint :)

**Comment by VirAlex** [comment]
Well done nice work, to me it's so fast I have no time to see skeleton 👍

File: src/components/searchresultpage/srpLaptop/SrpLaptop.jsx [hunk 4]
```diff
@@ -7,6 +7,7 @@ import { Configure, Index, Pagination } from 'react-instantsearch-hooks-web';
 import { useLocation } from 'react-router-dom';
 import { useRecoilValue } from 'recoil';
 // Import Components
+import SkeletonLoader from '../../hits/HitsSkeletonLoader';
```
Comment (VirAlex) [inline src/components/searchresultpage/srpLaptop/SrpLaptop.jsx:10]: You can remove it because it's imported twice 👍

**Review by VirAlex — CHANGES_REQUESTED** [review]

File: src/components/hits/components/HitsSkeletonLoader.jsx [hunk 5]
```diff
@@ -1,129 +1,37 @@
 import useScreenSize from '@/hooks/useScreenSize';
+import CustomSkeleton from '../skeletons/CustomSkeleton';
```
Comment (VirAlex) [inline src/components/hits/components/HitsSkeletonLoader.jsx:2]: You can use absolut import: 
'@/....'

More easy 👍

**Review by VirAlex — CHANGES_REQUESTED** [review]

**Comment by diegolarraz** [comment]
You really cannot see the skeleton - it's too fast, and for some reason the InjectedHits component does not mount the fallback component. It mounts on CustomHits and Facets. Very strange.

An improvement from the fade in animation though

**Review by matthew1809 — APPROVED** [review]
LGTM!

**Pull request merged**